client/webserver/site/src/localized_html/en-US/wallets.tmpl

```diff
@@ -1,154 +1,222 @@
+                      <img data-tmpl="quoteLogo">
+                      <span class="demi">
+                        <span data-tmpl="baseSymbol"></span>-<span data-tmpl="quoteSymbol"></span>
+                      </span>
+                    </td>
+                    <td class="d-none d-md-table-cell d-lg-none d-xxl-table-cell"><div data-tmpl="host" class="short-host"></div></td>
+                    <td>
+                      <span data-tmpl="price"></span>
+                      <span class="fs13 grey">
+                        <sup data-tmpl="priceQuoteUnit" class></sup>/<sub data-tmpl="priceBaseUnit"></sub>
+                      </span>
+                    </td>
+                    <td data-tmpl="volume">
+                      <span data-tmpl="volume"></span>
+                      <span data-tmpl="volumeUnit" class="fs15 grey"></span>
+                    </td>
+                  </tr>
+                </tbody>
+              </table>
+            </div>
+          </div>
+        </div>{{/* END MARKETS OVERVIEW */}}
+
+        {{- /* RECENT ORDERS */ -}}
+        <div class="col-24 col-xxl-8 d-flex stylish-overflow-lg border-top-lt-xxl invisible" id="orderActivityBox">
+          <div class="my-auto flex-center w-100 py-3" id="orderActivity">
+            <div class="d-flex align-items-start flex-column mw-90p">
+              <span class="fs28 sans-light mb-2">Recent <span data-asset-name=1></span> Activity</span>
+              <table id="ordersTable" class="my-auto" class="w-100">
+                <thead>
+                  <tr>
+                    <th>Trade</th>
+                    <th>Status</th>
+                    <th class="d-none d-md-table-cell">Filled</th>
+                    <th>Age</th>
+                    <th></th> {{/* Link */}}
+                  </tr>
+                </thead>
+                <tbody id="recentOrders">
+                  <tr id="recentOrderTmpl">
+                    <td data-tmpl="trade" class="text-nowrap">
+                      <span data-tmpl="fromQty"></span>
+                      <img data-tmpl="fromLogo" class="micro-icon">
+                      <span data-tmpl="fromSymbol" class="d-none d-md-inline"></span>
+                      <span>&rarr;</span>
+                      <span data-tmpl="toQty"></span>
+                      <img data-tmpl="toLogo" class="micro-icon">
+                      <span data-tmpl="toSymbol" class="d-none d-md-inline"></span>
+                    </td>
+                    <td data-tmpl="status"></td>
+                    <td data-tmpl="filled" class="d-none d-md-table-cell"></td>
+                    <td data-tmpl="age" class="text-nowrap"></td>
+                    <td><a class="ico-open fs14 pointer plainlink" data-tmpl="link"></a></td>
+                  </tr>
+                </tbody>
+              </table>
+            </div>
+          </div>
+          <div class="m-auto flex-center flex-column w-100" id="noActivity">
```

Comment:
This div looks better with a `py-3` like the `orderActivity` div that replaces it once there's activity.